api/helpers/utils.js

```diff
@@ -1,22 +1,13 @@
 'use strict';
 
-var _ = require('lodash');
+var { isArray, assignIn, each, compact, isEmpty } = require('lodash');
 var mongoose = require('mongoose');
 var clamav = require('clamav.js');
-var fs = require('fs');
```

Comment:
Where all of these not required??

Reply:
That's correct. They weren't used in this file. LUP as tons of these.